src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,157 @@
+  private final CANcoder canCoder = new CANcoder(DEVICE_ID_DEPLOY_CANCODER, CANIVORE_BUS_NAME);
+
+  private SysIdRoutine deployMotorSysIdRoutine = new SysIdRoutine(
+      new SysIdRoutine.Config(null, null, null, SysIdRoutineSignalLogger.logState()),
+      new SysIdRoutine.Mechanism((volts) -> deployMotor.setControl(voltageRequest.withOutput(volts.in(Volts))), null,
+          this));
+
+  private SysIdRoutine rollersMotorSysIdRoutine = new SysIdRoutine(
+      new SysIdRoutine.Config(null, null, null, SysIdRoutineSignalLogger.logState()),
+      new SysIdRoutine.Mechanism((volts) -> RollersMotor.setControl(voltageRequest.withOutput(volts.in(Volts))), null,
+          this));
+
+  public IntakeSubsystem() {
+    var intakeRollersConfig = new TalonFXConfiguration();
+    var intakeDeployConfig = new TalonFXConfiguration();
+    intakeRollersConfig.Slot0 = Slot0Configs.from(ROLLERS_SLOT_CONFIGS);
+    intakeRollersConfig.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+    RollersMotor.getConfigurator().apply(intakeRollersConfig);
+    intakeDeployConfig.Slot0 = Slot0Configs.from(DEPLOY_SLOT_CONFIGS);
+    deployMotor.getConfigurator().apply(intakeDeployConfig);
+    CANcoderConfiguration canCoderConfig = new CANcoderConfiguration();
+    canCoderConfig.MagnetSensor.AbsoluteSensorRange = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    canCoderConfig.MagnetSensor.SensorDirection = SensorDirectionValue.CounterClockwise_Positive;
+    canCoderConfig.MagnetSensor.MagnetOffset = 0.4;
+    canCoder.getConfigurator().apply(canCoderConfig);
+  }
+
+  private final VelocityTorqueCurrentFOC IntakeRollersMotorVelocity = new VelocityTorqueCurrentFOC(0, 0, 0, 1, false,
+      false,
+      false);
+
+  private final VelocityTorqueCurrentFOC IntakeDeployMotorVelocity = new VelocityTorqueCurrentFOC(0, 0, 0, 1, false,
+      false,
+      false);
+
+  /**
+   * Example command factory method.
+   *
+   * @return a command
+   */
+  public void intakeRollers(Measure<Voltage> volts) {
+    RollersMotor.setControl(voltageRequest.withOutput(volts.in(Volts)));
+  }
+
+  public void deploy(Measure<Voltage> volts) {
+    var intakeDeployConfig = new TalonFXConfiguration();
+    intakeDeployConfig.Slot0 = Slot0Configs.from(DEPLOY_SLOT_CONFIGS);
+    intakeDeployConfig.MotorOutput.Inverted = InvertedValue.CounterClockwise_Positive;
+    deployMotor.getConfigurator().apply(intakeDeployConfig);
+    deployMotor.setControl(voltageRequest.withOutput(volts.in(Volts)));
+  }
+
+  public void retractIntake(Measure<Voltage> volts) {
+    var intakeDeployConfig = new TalonFXConfiguration();
+    intakeDeployConfig.Slot0 = Slot0Configs.from(DEPLOY_SLOT_CONFIGS);
+    intakeDeployConfig.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+    deployMotor.getConfigurator().apply(intakeDeployConfig);
+    deployMotor.setControl(voltageRequest.withOutput(volts.in(Volts)));
+  }
+
+  // Spit means to reverse the motor direction to 'spit' the game piece out
```

Comment:
Instead of a comment to describe the meaning of the method name, just use a more descriptive name. Java method names can be up to 65535 characters (obviously try to keep them under 30ish)